site/install-rpm.xml

```diff
@@ -190,6 +193,29 @@ yum install erlang
         </p>
       </doc:subsection>
 
+      <doc:subsection name="install-from-suse-repository">
+        <doc:heading>Erlang packages from openSUSE</doc:heading>
+        <p>
+          Install Erlang directly using zypper as root:
+<pre class="sourcecode bash">
+zypper in erlang
```

Comment:
`sudo zypper install erlang` better to use the full subcommand instead of the abrev